bucketing/bucketing.go

```diff
@@ -27,11 +29,11 @@ type boundedHash struct {
 	BucketingHash float64 `json:"bucketingHash"`
 }
 
-func generateBoundedHashes(userId, targetId string) boundedHash {
+func generateBoundedHashes(bucketingKeyValue string, targetId string) boundedHash {
```

Comment:
```suggestion
func generateBoundedHashes(bucketingKeyValue, targetId string) boundedHash {
```

Multiple params of the same type only require you to define the type on the last param